src/adapters/TelnetAdapter.ts

```diff
@@ -360,24 +374,23 @@ export class TelnetAdapter {
         }
     }
 
-    private processUnhandledLines(responseText: string) {
```

Comment:
This is only called in one place. this function is only 1 line. so I moved the line to the callsite instead.